demo/src/app/components/+datepicker/demos/changed-value/changed-value.ts

```diff
@@ -0,0 +1,18 @@
+import { Component } from '@angular/core';
+import { DatePipe } from '@angular/common';
+
+@Component({
+  selector: 'demo-datepicker-changed-value',
+  templateUrl: './changed-value.html'
+})
+export class DemoDatepickerChangedValueComponent {
+  changedValue = false;
+  data: string;
+
+  constructor(private datePipe: DatePipe) {}
+
+  onValueChange(value: Event): void {
+    this.changedValue = true;
+    this.data = this.datePipe.transform(value);
```

Comment:
Is it necessary to use DatePipe llike that? Check this https://angular.io/guide/pipes#using-pipes